### DISCUSSION TITLE
Convert ArcGIS Min/Max Scales to Deck Zoom Level?

### DISCUSSION THREAD

**Discussion by jfuehner** [visgl/deck.gl#5566] [Q&A] [answered]
I am receiving min/max scale/ratio values (ie., 50000, 10000000) from an Arc service to be used to determine when to render objects within my deck layers. 

Would there be a recommended way to convert to convert these scale values to deck's corresponding zoom levels?

**Comment by Pessimistress** [comment, marked as answer]
Viewport scales are calculated as `Math.pow(2, zoom)`.
